Example/Example/Purchase/PurchaseLogicController.swift

```diff
@@ -145,21 +147,40 @@ final class PurchaseLogicController {
         name: "Standard",
         description: "3 - 5 days",
         shippingAmount: Money(amount: "0.00", currency: currencyCode),
-        orderAmount: Money(amount: "50.00", currency: currencyCode)
+        orderAmount: Money(amount: "50.00", currency: currencyCode),
+        taxAmount: Money(amount: "2.00", currency: currencyCode)
       ),
       ShippingOption(
         id: "priority",
         name: "Priority",
         description: "Next business day",
         shippingAmount: Money(amount: "10.00", currency: currencyCode),
-        orderAmount: Money(amount: "60.00", currency: currencyCode)
+        orderAmount: Money(amount: "60.00", currency: currencyCode),
+        taxAmount: Money(amount: "2.00", currency: currencyCode)
       ),
     ]
 
     let result: ShippingOptionsResult = .success(shippingOptions)
     commandHandler(.provideShippingOptionsResult(result))
   }
 
+  func selectShipping(shippingOption: ShippingOption) {
+    switch shippingOption.id {
+    case "standard":
```

Comment:
Suggestion: It might be more readable for a potential user browsing the Example app if, in this case, we opt for a simpler `if/else` clause rather than a `switch` with one case.

Reply:
addressed in 7a1290a